Easydict/NewApp/Configuration/Configuration+Defaults.swift

```diff
@@ -55,13 +55,18 @@ extension Defaults.Keys {
     static let clearInput = Key<Bool>("EZConfiguration_kClearInputKey", default: true)
     static let keepPrevResultWhenEmpty = Key<Bool>("EZConfiguration_kKeepPrevResultKey", default: true)
     static let selectQueryTextWhenWindowActivate = Key<Bool>("EZConfiguration_kSelectQueryTextWhenWindowActivate", default: false)
-    static let enableBetaNewApp = Key<Bool>("EZConfiguration_kEnableBetaNewAppKey", default: true)
 
     static let enableBetaFeature = Key<Bool>("EZBetaFeatureKey", default: false)
 
     static let appearanceType = Key<AppearenceType>("EZConfiguration_kApperanceKey", default: .followSystem)
     static let fontSizeOptionIndex = Key<UInt>("EZConfiguration_kTranslationControllerFontKey", default: 0)
     static let selectedMenuBarIcon = Key<MenuBarIconType>("EZConfiguration_kSelectedMenuBarIconKey", default: .square)
+
+    static let enableBetaNewApp = if #available(macOS 13.0, *) {
+        Key<Bool>("EZConfiguration_kEnableBetaNewAppKey", default: true)
+    } else {
+        Key<Bool>("EZConfiguration_kEnableBetaNewAppKey", default: false)
```

Comment:
This code looks a bit ugly, can it be optimized?

Reply:
Make a change just now, let me know if you have another suggestion.

Reply:
ok